packages/features/ee/billing/api/webhook/_customer.subscription.deleted.ts

```diff
@@ -46,6 +66,34 @@ const stripeWebhookProductHandler = (handlers: Handlers) => async (data: Data) =
   return await handler(data);
 };
 
+async function handlePhoneNumberSubscriptionDeleted(
+  subscription: Data["object"],
+  phoneNumber: { id: number }
+) {
+  if (!subscription.id) {
+    throw new HttpCode(400, "Subscription ID not found");
+  }
+
+  try {
```

Comment:
We don't seem to make any update to aiService for phone number in case of subscription deletion. It feels like something should happen there too?